package-lock.json

```diff
@@ -2891,13 +2891,13 @@
             }
```

Comment:
Please remove the changes to the `package-lock.json` file

Reply:
Does this mean I need to redo the whole process?

Reply:
I've removed the changes. 
Thanks.